x-pack/platform/packages/private/security/kbn_uiam_dev_cli/src/run_uiam.ts

```diff
@@ -85,27 +86,40 @@ services:
     environment:
       - AZURE_COSMOS_EMULATOR_PARTITION_COUNT=1
       - AZURE_COSMOS_EMULATOR_ENABLE_DATA_PERSISTENCE=false
-      - LOG_LEVEL=${env.UIAM_LOGGING_LEVEL.toLowerCase()}
+      - LOG_LEVEL=error
     healthcheck:
       test: ["CMD", "curl", "-sk", "https://127.0.0.1:8081/_explorer/healthcheck"]
       interval: 10s
       timeout: 5s
       retries: 30
       start_period: 20s
 
+  uiam-cosmosdb-init:
+    image: alpine:latest
+    network_mode: "service:uiam-cosmosdb-gateway"
+    depends_on:
+      uiam-cosmosdb-gateway:
+        condition: service_healthy
+    volumes:
+      - ${initScriptPath}:/init_cosmosdb.sh:ro
+    entrypoint: ["/bin/sh", "-c", "apk add --no-cache curl openssl && /bin/sh /init_cosmosdb.sh https://127.0.0.1:${env.UIAM_COSMOS_DB_GATEWAY_PORT}"]
+    restart: "no"
+
   uiam:
     image: docker.elastic.co/cloud-ci/uiam:${env.UIAM_GIT_REVISION}
     network_mode: "service:uiam-cosmosdb-gateway"
     depends_on:
       uiam-cosmosdb-gateway:
         condition: service_healthy
+      uiam-cosmosdb-init:
+        condition: service_completed_successfully
     volumes:
       - ${entrypointScriptPath}:/opt/jboss/container/java/run/run-java-with-custom-ca.sh
     entrypoint: /opt/jboss/container/java/run/run-java-with-custom-ca.sh
     environment:
       - quarkus.http.ssl.certificate.key-store-provider=JKS
       - quarkus.http.ssl.certificate.trust-store-provider=SUN
-      - quarkus.log.category."co".level=${env.UIAM_LOGGING_LEVEL}
+      - quarkus.log.category."co".level=DEBUG
```

Comment:
Is this change intentional, or leftover from debugging?

Reply:
Nice catch! Fixed https://github.com/elastic/kibana/pull/239713/commits/0412ec0d618f8cfa9b780210727b82f6630d3191